scripts/ostype.sh

```diff
@@ -6,7 +6,7 @@ if [ "${UNAME}" = "Darwin" ]; then
     echo "darwin"
 elif [ "${UNAME}" = "Linux" ]; then
     echo "linux"
-elif [[ "${UNAME}" == *"MINGW"* ]]; then
+elif [[ "${UNAME}" == *"MINGW"* ]] || [[ ${UNAME} == *"MSYS_NT"* ]]; then
```

Comment:
The reason for that is because when the travis windows shell is invoked, the `uname` matches `*"MSYS_NT"*`, however, if you'll look into the `.travis` file, you'll see that we've executing the underlying builder using the `$mingw64` shell. When running under the `mingw64` shell, the `uname` matches `*"MINGW"*`. I kept the `ostype.sh` to support both for the sole reason that we might want to detect the OS before invoking the `mingw64` shell.